### PR TITLE
Address test failures related to the freeipa-healthcheck 0.12 release

### PR DESCRIPTION
**Pull request by rcritten**
There are two problems:

1. A new report is included to demonstrate that the ipa-ca record is fine.
2. The error messages changed in IPADNSSystemRecordsCheck

### CODE REVIEW

File: ipatests/test_integration/test_ipahealthcheck.py [hunk 1]
```diff
@@ -1608,12 +1610,21 @@ def test_ipa_dns_systemrecords_check(self):
...
+            expected_msgs = {
+                "Expected SRV record missing",
+                "Got {count} ipa-ca A records, expected {expected}",
+                "Got {count} ipa-ca AAAA records, expected {expected}",
+                "Expected URI record missing",
+            }
+        else:
+            expected_msgs = {
+                "Expected SRV record missing",
+                "Unxpected ipa-ca address {server}",
```
Comment (flo-renaud) [inline ipatests/test_integration/test_ipahealthcheck.py:1624]: There is a typo and the exact message should be the same as https://github.com/freeipa/freeipa-healthcheck/blob/master/src/ipahealthcheck/ipa/idns.py#L219
```
Unexpected ipa-ca address {ipaddr}
```

**Review by flo-renaud — COMMENTED** [review]
@rcritten 
thanks for the PR, please find one inline comment

File: ipatests/test_integration/test_ipahealthcheck.py (hunk 1 above)
Comment (rcritten) [reply ipatests/test_integration/test_ipahealthcheck.py]: Sigh. I really need to be more careful. Fixed.

**Comment by flo-renaud** [comment]
@rcritten 
Thanks for the update, lgtm. You can remove the temp commit and I will ack when prci completes.

For future ref, link to successful runs:
- `fedora-latest/temp_commit`: [Details](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/47944142-a179-11ed-8cd2-fa163ee24cc1)
- `fedora-latest/temp_commit2`: [Details](http://freeipa-org-pr-ci.s3-website.eu-central-1.amazonaws.com/jobs/3d855afa-a17a-11ed-9f25-fa163ec98a23)

**Comment by rcritten** [comment]
Thanks, temp commit removed.

**Comment by flo-renaud** [comment]
master:

* bf9797f27b519049dc1462e399740d8b902dde9f tests: Add ipa_ca_name checking to DNS system records
* 6897ad997220b90f28d3b25d28b89344d5f88807 tests: Add new ipa-ca error messages to IPADNSSystemRecordsCheck

**Pull request closed without merging**